src/dsp_tools/resources/schema/data.xsd

```diff
@@ -83,23 +76,22 @@
         <xs:attribute name="permissions" type="xs:NCName" use="optional"/>
     </xs:complexType>
 
-    <!-- utf8 encoding value type -->
-    <xs:complexType name="utf8_encoding_text_type" mixed="true">
+    <!-- xml encoding value type -->
+    <xs:complexType name="xml_encoding_text_type" mixed="true">
         <xs:sequence>
             <xs:any processContents="skip" minOccurs="0" maxOccurs="unbounded"/>
         </xs:sequence>
-        <xs:attribute name="encoding" fixed="utf8" use="required"/>
+        <xs:attribute name="encoding" fixed="xml" use="required"/>
+        <xs:attribute name="resrefs" type="resrefs_type"/>
         <xs:attribute name="comment" type="xs:string"/>
         <xs:attribute name="permissions" type="xs:NCName" use="optional"/>
     </xs:complexType>
 
-    <!-- xml encoding value type -->
-    <xs:complexType name="xml_encoding_text_type" mixed="true">
+    <!-- encodingless text value type -->
+    <xs:complexType name="encodingless_text_type" mixed="true">
```

Comment:
no text is ever "encodingless" (which isn't a word either, I would argue), so maybe "unformatted" or "plain" would be better?

Reply:
ok, I missed the point of this change entirely... sure, in that case, unformatted would be absolutely the wrong choice.  
Understanding now, that you meant "encodingless" as in "the 'encoding' attribute is missing", this makes sense - but I still find it misleading. And I think the problem is that "utf-8" is indeed an encoding, but "xml" is not (XML is typically encoded as utf-8) to these two values don't really oppose each other, and the name of the attribute, "encoding" really only fits one of them. So the root cause is the bad naming there... And no matter what you do, as long as you refer to "encoding" I will find it misleading.  🙈 

The more I think about it, the more I doubt that it's a good idea to use the same type in the schema, here. Even if the validation rules are exactly the same, we have two different things at hand: `hasTitle` and `hasKeyword` are unformatted text (I wonder if you could enforce here, that it cannot have children?), `hasDescription` and `hasComment` are formatted text. So maybe the right thing to do would be to define two types `unformatted_text_type` and `formatted_text_type`, and to use the right type in the right place? Even if the validation is the same, conceptually they are different things.

Reply:
I like Balduins suggestion. This makes it also easier to change one type without creating changes in other places.

Reply:
Very good idea, it works like a charm: https://github.com/dasch-swiss/dsp-tools/pull/1101/commits/6bbdb349dd90cb3a71d4b1dde5b6969210db5ae7

And it is really possible to forbid children if it is an unformatted text :-)